pkg/client/clientset/versioned/typed/sriovnetwork/v1/fake/fake_sriovnetwork.go

```diff
@@ -20,9 +20,9 @@ type FakeSriovNetworks struct {
 	ns   string
 }
 
-var sriovnetworksResource = schema.GroupVersionResource{Group: "sriovnetwork", Version: "v1", Resource: "sriovnetworks"}
+var sriovnetworksResource = schema.GroupVersionResource{Group: "sriovnetwork.openshift.io", Version: "v1", Resource: "sriovnetworks"}
```

Comment:
how this change occurs?

Reply:
I manually changed the file, as it seems [hack/update-codegen.sh](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/hack/update-codegen.sh) is not aligned with the generated clientset.

Maybe it is worth creating a separate issue to get it working again.